login.php

```diff
@@ -8,6 +8,10 @@
 if (isset($_POST['signIn'])) {
   // Form was submitted
   $ldapDn = $_POST['inputDn'];
+  
+  //prefix and suffix can be added in config.inc.php
+  if (strpos($ldapDn,"=")!==false) $ldapDn = LOGIN_DN_PREFIX.$ldapDn.LOGIN_DN_SUFFIX;
```

Comment:
This is the wrong way round, isn't it? The dn should be expanded, when there is no '=' contained, i.e., `strpos($ldapDn,"=")===false`